strapi/design-system
# doc: update props descriptions for inputs in storybook

### What does it do?

- Added descriptions for props of `inputs` in storybook
- Updated dead links of `ViewSource`
- Updated descriptions and code example for `<Combobox createable />` as `CreatableCombobox` was removed 6 months ago

### Why is it needed?

For better reference and clarity 

### How to test it?

- clicking on `view source` button should navigate users to the correct file now
- all the `input` should have a clear table of props with descriptions (sometimes control) now

### Related issue(s)/PR(s)

DX-1707
